### DISCUSSION TITLE
Duda ejercicio 3

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#61] [Preguntas y respuestas] [answered]
Consulta:
¿Puedo usar un arreglo en el ejercicio 3 para almacenar el resultado de la multiplicacion?

**Comment by martinvilu** [comment]
Nope

**Reply by NicoHermosilla** [reply]
y como podria resolver un printf que tengo en mi funcion:
https://github.com/INGCOM-UNRN-P1/tp2-2023-NicoHermosilla
adjunto mi repositorio para que vea el ejercicio 3

**Reply by martinvilu** [reply]
¿Qué retornas? De lo que vimos ayer, ¿como se llamaban las funciones sin retorno?

**Reply by martinvilu** [reply]
La función en sí sería como una para el `hola mundo`, si ese ejercicio tuviese que ser implementado con una función.

**Reply by martinvilu** [reply, marked as answer]
Diapositiva 24
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/d6e62a7f-624d-47a7-acee-1adafe810d0f)
